README.markdown

```diff
@@ -25,6 +25,7 @@ Table of Contents
     * [resty.core.misc](#restycoremisc)
     * [resty.core.time](#restycoretime)
     * [resty.core.worker](#restycoreworker)
+    * [resty.core.semaphore](#restycoresemaphore)
```

Comment:
This does not make sense since we do not have a `resty.core.semaphore` module any more. Now it's `ngx.semaphore`.